aws/test/colored-money-hardhat/test/RLN.test.ts

```diff
@@ -37,42 +74,42 @@ describe('Test RLN functionality', async function () {
   });
 
   it('Deposits', async () => {
-    const pendingDepositsInitial = await user.checkPendingDeposits();
-    let balanceInitial = 0;
-    try {
-      balanceInitial = pendingDepositsInitial[UserConfig.CONTRACT_ADDRESS.toLowerCase()][1].balance;
-    } catch (error) {
-      // No initial pending balance
-    }
-    try {
-      // Make deposit
-      const txReceipts = await user.makeDeposit({
-        tokenContractAddress: UserConfig.CONTRACT_ADDRESS,
-        value: valueDeposit,
-        tokenId: entityId,
-        feeWei: '0',
-      });
-      // Check we have transaction hashes in Nightfall
-      expect(txReceipts.txReceipt.transactionHash).to.a('string').and.to.include('0x');
-      expect(txReceipts.txReceiptL2.transactionHash).to.a('string').and.to.include('0x');
-      expect(user.nightfallDepositTxHashes[0]).to.a('string').and.to.include('0x');
-    } catch (error) {
-      console.log('Error in deposit', error);
+    const balanceInitial = await getRLNBalance(user, UserConfig.CONTRACT_ADDRESS, entityId);
+    for (let i = 0; i < 2; i++) {
+      try {
+        // Make deposit
+        const txReceipts = await user.makeDeposit({
+          tokenContractAddress: UserConfig.CONTRACT_ADDRESS,
+          value: valueDeposit,
+          tokenId: entityId,
+          feeWei: '0',
+        });
+        // Check we have transaction hashes in Nightfall
+        expect(txReceipts.txReceipt.transactionHash).to.a('string').and.to.include('0x');
```

Comment:
(?)

```suggestion
        expect(txReceipts.txReceipt.transactionHash).to.be('string').and.to.include('0x');
```

Reply:
This is icorrect. It's not `be` because it will compare with the value. I need to compare with the type and need `a`